docs/plugins/spectate.rst

```diff
@@ -2,16 +2,50 @@ spectate
 ========
 
 .. dfhack-tool::
-    :summary: Automatically follow exciting dwarves.
+    :summary: Automatically follow productive dwarves.
     :tags: fort interface
-    :no-command:
 
 Usage
 -----
 
 ::
 
     enable spectate
+    disable spectate
```

Comment:
for better or worse, I've only documented the enable form for all the other commands. you can take this line out to bring it in line with the rest of the documentation.